### PR TITLE
Add script to change times on input

### LINKED ISSUE #826: Autofill last-date-to-submit and grading-deadline when setting due date
The last-date-to-submit and grading-deadline fields no longer automatically update when a due date is set. It used to be that these would update to be the same as the due date unless they had already been set to be later than that, which was very convenient.

### PR DESCRIPTION
**Pull request by nayak16**
Fixes #826

### CODE REVIEW

File: app/assets/javascripts/init_handin_datetimepickers.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      if (grading_deadline_pickr.selectedDates[0].getTime() < cur_date.getTime()) {
+        grading_deadline_pickr.setDate(cur_date, true);
+      }
+
+      if (end_at_pickr.selectedDates[0].getTime() < cur_date.getTime()) {
+        end_at_pickr.setDate(cur_date, true);
+      }
+
+      // Reinstall handler
+      flatpickr_inst.set('onClose', [onCloseHandler])
```
Comment (zyx-billy) [inline app/assets/javascripts/init_handin_datetimepickers.js]: Everything looks good to me. Though I don't think we need to reinstall handler each time? I tried without this line and it still worked multiple times.

**Comment by zyx-billy** [comment]
Things worked perfectly. Do you think we should add a similar handler to watch the opposite direction? i.e. when end_at decreases to before due_at, we update due_at. And similarly when start_at increases too much we update everything else.

File: app/assets/javascripts/init_handin_datetimepickers.js (hunk 1 above)
Comment (nayak16) [inline app/assets/javascripts/init_handin_datetimepickers.js]: What browser did you try it on? I tried on Chrome without this and it wouldn't work multiple times

**Comment by nayak16** [comment]
That would be good but I'm trying to figure out a more generalizable approach rather than hard coding all the restrictions inside the handler.

File: app/assets/javascripts/init_handin_datetimepickers.js (hunk 1 above)
Comment (zyx-billy) [inline app/assets/javascripts/init_handin_datetimepickers.js]: I tried it on Firefox yesterday and I just tried on Chrome, both worked for me. Perhaps try adding a log statement in the handler and see if the handler was called?

**Pull request merged**